<p>This rule applies whenever the nested if statement is used . </p>

<p>It is good programming practice to avoid nested IF statements: use ELSEIF or CASE WHEN clauses to get quicker drop-out. </p>
<h2>Noncompliant Code Example</h2>
<pre>
	IF FALSE THEN
			IF FALSE THEN
				IF FALSE THEN
					IF FALSE THEN 
						SET OutputRoot=InputRoot;
					END IF;
</pre>
<h2>Compliant Solution</h2>
<pre>
	ELSEIF A = B THEN 
	  ELSEIF FALSE THEN 
		 ELSEIF end THEN
</pre>