<p>All binary operators should be separated from their operands by spaces. </p>

<p>It is good programming practice to give space around binary operators. </p>
<h2>Noncompliant Code Example</h2>
<pre>
	WHEN versionRef>'1.1' THEN
</pre>
<h2>Compliant Solution</h2>
<pre>
	WHEN versionRef > '1.1' THEN
</pre>