<p>This rule applies whenever the CASE Statement having single when in the esql, it should be replaced by IF Statement. </p>

<p>Replace this "CASE" statement by "if" statements to increase the code readeablity. </p>
<h2>Noncompliant Code Example</h2>
<pre>
	CASE size
		  WHEN minimum + 0 THEN
		    SET description = 'small';
		    END CASE;
</pre>
<h2>Compliant Solution</h2>
<pre>
	CASE size
		  WHEN minimum + 0 THEN
		    SET description = 'small';
		  WHEN minimum + 1 THEN
		    SET description = 'medium';        
		 END CASE;
</pre>