<p>This rule applies to check Cyclomatic Complexity </p>
<p>Cyclomatic Complexity should not be higher than the given threshold value .</p>

<h2>Noncompliant Code Example</h2>
<pre>
	 Cyclomatic complexity should not be greater than 10
</pre>
<h2>Compliant Solution</h2>
<pre>
	Cyclomatic complexity should be  below  10
</pre>