<p>This rule applies whenever the EXTERNAL variable is declared but not given initial value . </p>

<p>It is good programming practice to give an EXTERNAL variable an initial value. </p>
<h2>Noncompliant Code Example</h2>
<pre>
	DECLARE deployEnvironment EXTERNAL CHARACTER '';
</pre>
<h2>Compliant Solution</h2>
<pre>
	DECLARE deployEnvironment EXTERNAL CHARACTER 'dev';
</pre>