<p>Each ESQL file should contains header information as below:</p>
<h2>Noncompliant Code Example</h2>
<pre>
	/*
	Change Log:
	S.No. Incident Number  Jira Number  Description  Date Fixed  Updated 
	
	*/
</pre>
<h2>Compliant Solution</h2>
<pre>
	/*
	Change Log:
	S.No. Incident Number  Jira Number  Description  Date Fixed  Updated ESQL/Node
	1       INC0182018        DI-999       Test defect  09/01/2017  test.esql
	*/
</pre>