<p>Filter nodes that only have 1 return are not providing filtering to more then one available path. </p>
<p>A filter node with a single return could be either a logic error or could be redundant. </p>
<h2>Noncompliant Code Example</h2>
<pre>
	CREATE FILTER MODULE Flow2_Filter
      CREATE FUNCTION Main() RETURNS BOOLEAN
      BEGIN
        SET OutputRoot.XMLNSC.Response.details.person.age = '20';
       RETURN FALSE;
      END;
     END MODULE;
</pre>
<h2>Compliant Solution</h2>
<pre>
	CREATE FILTER MODULE Flow2_Filter
      CREATE FUNCTION Main() RETURNS BOOLEAN
      BEGIN
         RETURN FALSE;
      END;
      CREATE FUNCTION Main() RETURNS BOOLEAN
      BEGIN
         RETURN FALSE;
      END;
     END MODULE;
</pre>