<p>This rule applies to check the length of ESQL function or procedure</p>
<p>The length of the function or procedure should not be higher than the given threshold value .</p>

<h2>Noncompliant Code Example</h2>
<pre>
	 Function or Procedure longer than 150 code lines.
</pre>
<h2>Compliant Solution</h2>
<pre>
	Function or Procedure shorter than 150 code lines.
</pre>