<p>This rule applies whenever the <code>keyword</code> is used in the esql, it should be in the UPPERCASE. </p>

<p>The requirement for UPPERCASE for all the keywords in esql helps to identify the keywords hence improves readability. </p>
<h2>Noncompliant Code Example</h2>
<pre>
	Create COMPUTE MODULE Compute1
</pre>
<h2>Compliant Solution</h2>
<pre>
	CREATE COMPUTE MODULE Compute1
</pre>