<p>This rule applies whenever the Variable or counter is declared . </p>

<p>It is good programming practice to give meaningful names to variable and counters </p>
<h2>Noncompliant Code Example</h2>
<pre>
	DECLARE x CHARACTER 'asfsd';
</pre>
<h2>Compliant Solution</h2>
<pre>
	DECLARE wrong CHARACTER 'asfsd';
</pre>