<p>This rule applies whenever a <code>SET</code> statement is written, it should contain </p>
<p>spaces around equal sign.</p>
<p>The requirement for spaces around equal sign in <code>SET</code> statement improves readability. </p>
<h2>Noncompliant Code Example</h2>
<pre>
	SET OutputRoot=InputRoot;
</pre>
<h2>Compliant Solution</h2>
<pre>
	SET OutputRoot = InputRoot;
</pre>